manifests/n/Nicotine+/Nicotine+/3.3.7/Nicotine+.Nicotine+.locale.en-US.yaml

```
@@ -0,0 +1,59 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Nicotine+.Nicotine+
+PackageVersion: 3.3.7
+PackageLocale: en-US
+Publisher: Nicotine+

```

```suggestion
Publisher: Nicotine+ Team
```
This should match control panel as closely as possible